.travis.yml

```diff
@@ -1,37 +1,36 @@
 language: scala
 
+jdk:
+  - openjdk8
+  - openjdk11
```

Comment:
This might be a bit too confusing, as OpeJDK is not used in the end?

Reply:
That's correct.  I'll add a note.